demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -585,6 +586,20 @@ def test_handle_all_custom_content_flag(self, tmp_path):
             custom_content_names = [cco["name"] for cco in env.CUSTOM_CONTENT]
             assert ordered(custom_content_names) == ordered(downloader.input_files)
 
+    def test_handle_init_flag(self, tmp_path, mocker):
+        env = Environment(tmp_path)
+        mocker.patch.object(builtins, "input", return_value="test_pack_name")
```

Comment:
please save it to variable, and later make sure it was only called once (hint: `called_once`)